paddlenlp/transformers/tokenizer_utils_base.py

```diff
@@ -1738,7 +1743,55 @@ def save_resources(self, save_directory):
             src_path = self.init_kwargs[name]
             dst_path = os.path.join(save_directory, file_name)
             if os.path.abspath(src_path) != os.path.abspath(dst_path):
-                copyfile(src_path, dst_path)
+                shutil.copyfile(src_path, dst_path)
+
+    def save_to_hf_hub(
+        self,
+        repo_id: str,
+        private: Optional[bool] = None,
+        commit_message: Optional[bool] = None,
+        revision: Optional[str] = None,
+        create_pr: bool = False,
+    ):
+        """
+        Uploads all elements of this tokenizer to a new HuggingFace Hub repository.
+        Args:
+            repo_id (str): Repository name for your model/tokenizer in the Hub.
+            private (bool, optional): Whether the model/tokenizer is set to private
+            commit_message (str, optional) — The summary / title / first line of the generated commit. Defaults to: f"Upload {path_in_repo} with huggingface_hub"
+            revision (str, optional) — The git revision to commit from. Defaults to the head of the "main" branch.
+            create_pr (boolean, optional) — Whether or not to create a Pull Request with that commit. Defaults to False.
+                If revision is not set, PR is opened against the "main" branch. If revision is set and is a branch, PR is opened against this branch.
+                If revision is set and is not a branch name (example: a commit oid), an RevisionNotFoundError is returned by the server.
+
+        Returns: The url of the commit of your model in the given repository.
+        """
+        token = HfFolder.get_token()
+        if token is None:
+            raise ValueError(
+                "You must login to the Hugging Face hub on this computer by typing `transformers-cli login`."
+            )
+
+        repo_url = create_repo(repo_id, token=token, private=private, exist_ok=True)
```

Comment:
Same as above. Checking for `token` is not needed.

```suggestion
        repo_url = create_repo(repo_id, private=private, exist_ok=True)
```